NDTensors/ext/NDTensorscuTENSORExt/contract.jl

```diff
@@ -0,0 +1,32 @@
+using NDTensors:
+  NDTensors,
+  DenseTensor,
+  Tensor,
+  array,
+  contract!,
+  contraction_output,
+  contraction_output_type,
+  contract_inds,
+  dense,
+  inds
```

Comment:
Seems like not all of these are needed.

Reply:
Thanks I missed this cleanup. I removed the unused functions